docs/assistant/security-assistant.asciidoc

```diff
@@ -177,4 +177,17 @@ To enable the knowledge base:
 . Turn on the *Knowledge Base* option.
 . Click *Save*. The knowledge base is now active.
 +
-When the knowledge base is active, a quick prompt for {esql} queries becomes available. It provides a good starting point for your {esql} conversations and questions.
\ No newline at end of file
+When the knowledge base is active, a quick prompt for {esql} queries becomes available. It provides a good starting point for your {esql} conversations and questions.
+
+[discrete]
+[[ai-assistant-queries]]
+### How to get the most from your queries
+
+The goal of the AI Assistant is to help users take full advantage of the Elastic Security platform to improve their cyber operations. Its ability to assist you depends on the specificity and detail of your questions. The more context and detail you provide, the more tailored and useful its responses will be. 
+
+To maximize its usefulness, consider using more detailed or elaborate prompts. For instance, after asking for an {esql} query example, you could ask a follow-up question like, “Could you give me some other examples and provide comments explaining the query you just gave?”. You can also ask for clarification or further exposition. 
+
+In addition to practical advice, AI Assistant can offer conceptual advice, tips, and best practices for enhancing your security measures. You can ask it, for example:
+
+. “How do I set up a machine learning job in Elastic Security to detect anomalies in network traffic volume over time?”
+. “I need to monitor for unusual file creation patterns that could indicate ransomware activity. How would I construct this query using EQL?”
```

Comment:
A bulleted list is more appropriate here, because these aren't steps being performed in a certain order.
```suggestion
* “How do I set up a machine learning job in Elastic Security to detect anomalies in network traffic volume over time?”
* “I need to monitor for unusual file creation patterns that could indicate ransomware activity. How would I construct this query using EQL?”
```